### DISCUSSION TITLE
Any Admins PLS ?

### DISCUSSION THREAD

**Discussion by SebastienXia** [community/community#111562] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

# An admin to manage those who spam ? it becomes nonsense

**Comment by ghostinhershell** [comment]
Hi @SebastienXia, 

We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Reply by peterldowns** [reply]
@ghostinhershell piggybacking on this issue, please consider doing something about spam in the support forum. I recently logged in for the first time in a while and on the default forum home page I was shown a large number of obvious spam links. Here is a screen shot, but you can view them by visiting

https://github.com/orgs/community/discussions

<img width="864" alt="image" src="https://github.com/community/community/assets/824173/9656283b-1d40-4cc6-bdcf-694b5c78dd02">

Please institute a basic spam filter. I reported [one of these posts](https://github.com/orgs/community/discussions/113984), but it's surprising to me that they exist at all given Github's access to machine learning engineers.
 
<img width="636" alt="image" src="https://github.com/community/community/assets/824173/97293d04-a3e7-483e-b8d5-405ae7315e3f">

**Reply by ettaboyle** [reply]
Thanks for this comment, @peterldowns & @SebastienXia  ♥️ We work closely with internal teams on ML models that detect "bad actor" "spammy" and "off-topic" patterns in both discussions and comments. They help us to triage these at scale, but bots and gonna bot ... Patterns change and a lot of this requires a human to review. 

Trust us when we say we are frustrated by it just as much as you (and so many of our users). We've added more folks to the team to help manage off topic & spam posts across all categories. This is reactive and doesn't prevent the problem, but we are actively working to solve this at scale. 

The last note I want to share is that we have seen the bulk of these posts in this accessibility category. Our best guess is that it is because it comes first alphabetically. We appreciate you calling this out.

Thank you for your quality contributions! Its really why we all do what we do. We want to curate a safe space where users can help users, find inspiration, and provide insight and product feedback to our teams. 

We remain vigilant. Thank you! ✨

**Reply by ghost** [reply]
@ettaboyle @ghostinhershell, I understand that my issue is not directly related to the topic of this discussion (except of the word "spam"), but could you please look at this issue: https://github.com/orgs/community/discussions/121640. I have tried contacting github support, but all of my tickets are closed instantly without an explanation.

Maybe you could tell me what to do to get help with this..

**Comment by cehfisher** [comment]
Thanks for posting in the GitHub Community, @SebastienXia! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've asked @queenofcorgis to move this issue into the correct category for you. Good luck!
